kinetic/distribution.yaml

```diff
@@ -10063,10 +10063,12 @@ repositories:
       - rdl_cmake
       - rdl_dynamics
       - rdl_urdfreader
+      - rdl_msgs
```

Comment:
All packages need to be in alphabetical order to keep lists deterministic in ordering for cleaner merging.

Reply:
ah yes my fault, got it updated